### PR TITLE
docs: add info for smart builds env vars

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

### CODE REVIEW

File: src/content/reference/cli.mdx [hunk 1]
```diff
@@ -111,6 +111,8 @@ This command builds a multi-architecture Docker image named *myapp* with the lat
 By using these commands, you can easily build the application image for different platforms without needing to maintain separate Dockerfiles or perform manual modifications.
 This is particularly useful when you want to deploy your application to heterogeneous environments where you have both x86_64 and ARM-based devices, such as a mixed-cluster Kubernetes setup.
 
+Okteto incorporates a smart build mechanism that leverages caching to prevent redundant builds for identical commits within your project. This optimization strategy minimizes build times and resource utilization. To deactivate the smart build feature, set the environment variable OKTETO_SMART_BUILDS_ENABLED to the desired value. This provides users with the flexibility to control whether Okteto should utilize smart builds. In cases where your project manifests include multiple builds, and you wish to rebuild only those modified in your latest commit, utilize the OKTETO_SMART_BUILDS_USING_BUILD_CONTEXT environment variable. This functionality enables the targeted rebuilding of specific project components based on recent changes.
```
Comment (pchico83) [inline src/content/reference/cli.mdx:114]: @jLopezbarb let's keep it undocumented until we document how it works.

**Comment by maroshii** [comment]
This needs to be moved to `src/content/reference/okteto-cli.mdx`

**Comment by omnomagonz** [comment]
@pchico83 should we close this PR since we plan to document this in the new `Okteto manifest` > `build` section?

**Comment by pchico83** [comment]
> @pchico83 should we close this PR since we plan to document this in the new `Okteto manifest` > `build` section?

@omnomagonz makes sense

**Comment by omnomagonz** [comment]
Closing in favor of a future PR to address this content

**Pull request closed without merging**